jobs/list.ts

```diff
@@ -36,4 +37,7 @@ export const allJobs: CSCronJob[] = [
     { cronTime: '00 00 10 01 * *', jobFunction: postStatisticsToSlack, name: 'postStatisticsToSlack' },
     // Every night, trigger actions for courses that ended yesterday
     { cronTime: '00 00 10 * * *', jobFunction: notificationsEndedYesterday, name: 'notificationsEndedYesterday' },
+    // Revalidate streaks are used to revalidate if streaks are still meeting their conditions and updates them accordingly
+    { cronTime: '00 00 02 * * 1', jobFunction: () => revalidateStreaks(StreakTimeFrame.WEEK), name: 'weeklyStreakRevalidation' },
```

Comment:
so this mean every monday at 2 a.m. or? Just a question for me to understand the time 😄

Reply:
yes This means every first day of a week ( last number ) at 02 AM